src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -86,82 +81,63 @@ for further details.
   }
 }
 
-.mainAreaAndFooter {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-}
-
-/**
- * Main area row
- */
-
-.mainArea {
-  display: flex;
-  flex-grow: 1;
-  gap: var(--sidecar-gap);
-  min-width: 0;
-  justify-content: center;
-  padding-bottom: 32px;
-  padding-top: var(--main-area-top-padding);
+.contentWrapper {
   padding-left: var(--main-area-padding-left);
   padding-right: var(--main-area-padding-right);
+  padding-top: 32px;
+  width: 100%;
+
+  @media (--dev-dot-desktop) {
+    max-width: calc(100vw - var(--dev-dot-sidebar-width));
+  }
 }
 
 .main {
-  align-self: flex-start;
-  flex-grow: 1;
-  min-width: 0;
-
-  /* 50vh padding-bottom ensures that all
-  jump-nav items have room to be jumped to */
+  /* Ensure all TableOfContents items have room to be jumped to */
   padding-bottom: 50vh;
-
-  @media (--dev-dot-desktop) {
-    max-width: var(--main-area-max-width);
-  }
 }
 
 .editOnGithubLink {
   margin-top: 64px;
 }
 
-.sidecar {
-  display: none;
-
-  @media (--dev-dot-desktop) {
-    align-self: flex-start;
-    display: block;
-    flex-shrink: 0;
-    position: sticky;
-    top: calc(var(--sticky-bars-height) + var(--main-area-top-padding));
-    width: var(--sidecar-width);
-  }
+.footerAreaWrapper {
+  margin-top: 106px;
 }
 
-/**
- * Footer area row
- */
+/*
+***
+* The following classes are grouped in a way that ensures the footer is always
+* left-aligned and the same width as the main content area.
+***
+*/
```

Comment:
🗒️ Since the border is going away with [Update footers to align with new designs](https://app.asana.com/0/1201987349274776/1202074017129359/f), I went ahead and removed in this PR. I'll start a PR for doing the rest of the Footer changes today or tomorrow! Changes here: https://github.com/hashicorp/dev-portal/pull/442/commits/1faddf46a8d76c2c183855a94d91526e6c1873c0.